packages/plugin-ext/src/plugin/plugin-context.ts

```diff
@@ -1223,9 +1235,27 @@ export function createAPIFactory(
             /** @stubbed MappedEditsProvider */
             registerMappedEditsProvider(documentSelector: theia.DocumentSelector, provider: theia.MappedEditsProvider): Disposable {
                 return Disposable.NULL;
+            },
+            /** @stubbed ChatRequestHandler */
+            createChatParticipant(id: string, handler: theia.ChatRequestHandler): theia.ChatParticipant {
+                return {
+                    id,
+                    requestHandler: (request: theia.ChatRequest, context: theia.ChatContext, response: theia.ChatResponseStream, token: CancellationToken) => { },
```

Comment:
Why not just assign the passed-in request handler instead of a dummy?

Reply:
Done in new commit